completions/cvs

```diff
@@ -43,9 +43,9 @@ _comp_xfunc_cvs_compgen_roots()
     local -a cvsroots=()
     [[ -v CVSROOT ]] && cvsroots=("$CVSROOT")
     [[ -r ~/.cvspass ]] && cvsroots+=($(awk '{ print $2 }' ~/.cvspass))
-    [[ -r CVS/Root ]] && mapfile -tO ${#cvsroots[@]} cvsroots <CVS/Root
+    [[ -r CVS/Root ]] && mapfile -tO "${#cvsroots[@]}" cvsroots <CVS/Root
```

Comment:
I wonder/forget what's the reason for this change, the length can never be anything but an integer, right?

Reply:
This is based on the [quoting rule](https://github.com/scop/bash-completion/blob/master/doc/styleguide.md#quoting-of-words). These are the background discussions:

- https://github.com/scop/bash-completion/pull/931#discussion_r1174403080
- https://github.com/scop/bash-completion/issues/932
- https://github.com/scop/bash-completion/pull/986

More explicitly, `IFS` can contain an integer. Even if such an `IFS` is unusual, there is no reason to leave it unquoted when just quoting can avoid it. For this reason, the unquoted arguments of commands have always been a red flag for me, and it is annoying that I stumble upon the line every time I glance at the code.